yellowbrick/features/rankd.py

```diff
@@ -166,11 +207,6 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
-
-        # TODO: This class is identical to the Parallel Coordinates version,
```

Comment:
As far as I can tell, yes.  ParallelCoordinates and RadViz now inherit their `fit` methods from `features.base.DataVisualizer`, but the functionality there is not applicable to RankD. 

From a quick scan over the other modules in the `features` package, I don't see any others that do the same thing on `fit` that we're doing here, so there's probably no need to hoist this higher for now.